spec/features/navigation_spec.rb

```diff
@@ -108,4 +127,118 @@
       expect(page).to have_content("The page you were looking for doesn't exist (404)")
     end
   end
+
+  describe "as a user i see all visitor links plus" do
+    context "i see links to /profile and /logout and" do
+      context "i do not see links to /login and /register and" do
+        it "i see 'Logged in as <username>'" do
+
+          user = User.create!(
+            name: "Steve",
+            address:"123 Main St.",
+            city: "Fort Collins",
+            state: "GA",
+            zip: "66666",
+            email: "chunky_lover@example.com",
+            password: "123password"
+          )
+
+          visit "/login"
+          fill_in :Email, with: user.email
+          fill_in :Password, with: user.password
+          click_button "Login"
+
+          within 'nav' do
+            expect(page).to have_link('Home')
+            expect(page).to have_link('All Items')
+            expect(page).to have_link('All Merchants')
+            expect(page).to have_link('Cart: 0')
+            expect(page).to have_link('Profile')
+            expect(page).to have_link('Logout')
+            expect(page).to have_no_link('Register')
+            expect(page).to have_no_link('Login')
+            expect(page).to have_content("Logged in as #{user.name}")
+          end
+
+        end
+      end
+    end
+  end
+
+  describe "as a merchant, i see all user links" do
+    it "plus a link to my merchant dashboard" do
+
+      merchant = User.create!(
+        name: "Steve",
+        address:"123 Main St.",
+        city: "Fort Collins",
+        state: "GA",
+        zip: "66666",
+        email: "chunky_lover@example.com",
+        password: "123password",
+        role: 1
+      )
+
+      visit "/login"
+      fill_in :Email, with: merchant.email
+      fill_in :Password, with: merchant.password
+      click_button "Login"
+
+      within 'nav' do
+        expect(page).to have_link('Home')
+        expect(page).to have_link('All Items')
+        expect(page).to have_link('All Merchants')
+        expect(page).to have_link('Cart: 0')
+        expect(page).to have_link('Profile')
+        expect(page).to have_link('Logout')
+        expect(page).to have_no_link('Register')
+        expect(page).to have_link('Dashboard')
+        expect(page).to have_no_link('Login')
+        expect(page).to have_content("Logged in as #{merchant.name}")
+      end
+    end
+  end
+
+  describe "As an admin,", type: :feature do 
+    it "I see the same links as a regular user plus /admin and /admin/user minus /cart." do 
+      admin = User.create!(
+        name: "Steve",
+        address:"123 Main St.",
+        city: "Fort Collins",
+        state: "GA",
+        zip: "66666",
+        email: "chunky_lover@example.com",
+        password: "123password",
+        role: 2
+      )
+
+      visit "/login"
+      fill_in :Email, with: admin.email
+      fill_in :Password, with: admin.password
+      click_button "Login"
+
+      within 'nav' do
```

Comment:
do we need to click the ones that work or should we do that on later tests

Reply:
Hmm, I think we could go either way with this.  The links we added - profile, merch dash, admin dash, and see all users - have either already been tested elsewhere (profile, merch dash, admin dash redirects in US15) and see all users will be built out in US53.  But it also kind of makes sense to make sure the nav bar functionality of these links work, too.

I can add the link tests for nav in my US21 work, not a very heavy lift to add that.